docs/estimators-ti.rst

```diff
@@ -66,6 +67,12 @@ In addition to the free energy differences, we also have access to the errors on
     0.75  0.015038  0.011486  0.007458  0.000000  0.006447
     1.00  0.016362  0.013172  0.009858  0.006447  0.000000
 
+If the simulations are performed at certain gaussian quadrature points, :class:`~alchemlyb.estimators.TI_GQ` can be used to estimate the free energy.
+The usage of :class:`~alchemlyb.estimators.TI_GQ` is similar to :class:`~alchemlyb.estimators.TI`, but instead of the free energy differences between 
+each :math:`\lambda` window, the values in the ``delta_f_`` and ``d_delta_f_`` tables are cumulative addition of estimation from one :math:`\lambda` window to another.
+To be consistent with :class:`~alchemlyb.estimators.TI` and other estimators, the diagonal values are set to zeros and two end states at :math:`\lambda`
+0 and 1 are added, although the simulation may not be performed at :math:`\lambda` 0 and 1. However, they are not considered to estimate the final results.
+For example, the values at :math:`\lambda` 0 is 0 and :math:`\lambda` 1 is the same as the previous gaussian quadrature point.
```

Comment:
What do you mean `However, they are not considered to estimate the final results.`, I was under the impression that the for lambda points `[0.1127, 0.5, 0.88729]`, the result at point `0.88729` is the free energy difference between 0 and 1?

`:math:`\lambda` 0 is 0` is not the same as the previous gaussian quadrature point`

Reply:
> What do you mean `However, they are not considered to estimate the final results.`

Sorry for the confusion. I meant that they didn't contribute to the final free energy values, as there were no estimations at lambda 0 and 1. I have removed this sentence. The next sentence should explain this.